android-agent/src/test/java/io/opentelemetry/android/initialization/TestInitializationEvents.kt

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.android.initialization
+
+import io.opentelemetry.android.config.OtelRumConfig
+import io.opentelemetry.sdk.trace.export.SpanExporter
+
+class TestInitializationEvents : InitializationEvents {
```

Comment:
Is this the same as the noop version? Can we reuse that instead?

Reply:
It's kinda the same, though the reason I created it was for [this test](https://github.com/open-telemetry/opentelemetry-android/pull/486/files#diff-e8aa70e2db0e749932aacc44941988c9754c369a96aba4d3afb02ea8e8505756R21). So that the test can verify that the `InitializationEvents.get()` function does search for implementations in the classpath. If no implementation is found, then the function returns the noop impl, so that's why I couldn't use the same one for the test.

Reply:
ok, I think that makes sense then.